logging/simplelog/simplelog.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample simplelog writes some entries, lists them, then deletes the log.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	// [START imports]
+	// NOTE: This will become cloud.google.com/go/logging soon.
+	"cloud.google.com/go/preview/logging"
+
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	// [END imports]
+)
+
+func main() {
+	if len(os.Args) != 3 {
+		if len(os.Args) == 2 {
+			usage("Missing command.")
+		} else {
```

Comment:
Actually, let's make this easier.

if len(os.Args) == 2 {
   usage("...")
}

if len(os.Args) != 3 {
    usage("")
}